pkg/Library/NodeGraph/NodeInspector.js

```diff
@@ -180,10 +180,11 @@ async constructConnections(node, {graph, nodeTypes, candidates}, service) {
 async renderBinding(node, name, candidates, graph, nodeTypes, service) {
   if (candidates) {
     const froms = candidates.map(candidate => this.renderCandidate(candidate, graph)).filter(from => from);
-    const value = node.connections?.[name] || [];
+    const selected = node.connections?.[name] || [];
```

Comment:
did you intentionally restore this file? i changed it yesterday as part of the connection de/composing ({from, storeName} => `${from}:${storeName})

Reply:
I saw that and thought it looked like I messed up somewhere ... sorry :(